packages/commonwealth/client/scripts/views/components/Comments/CreateComment.tsx

```diff
@@ -108,9 +110,18 @@ export const CreateComment = ({
 
   // token balance check if needed
   const tokenPostingThreshold: BN = app.chain.getTopicThreshold(activeTopic.id);
-
+  console.log(app.user.activeAccount?.tokenBalance);
+  if (tokenPostingThreshold && !balanceLoading) {
```

Comment:
Can we fix this initialization flow? conditionals / setting right at the root of the component function is not properly reactive.

Reply:
updated to useEffect